plugins/wazo-snom/10.1.119.10/pkgs/pkgs.db

```diff
@@ -0,0 +1,135 @@
+[pkg_315-fw]
+description: Firmware for Snom D315
+description_fr: Firmware pour Snom D315
+version: 10.1.119.10
+files: 315-fw
+install: snom-fw
+
+[pkg_335-fw]
+description: Firmware for Snom D335
+description_fr: Firmware pour Snom D335
+version: 10.1.119.10
+files: 335-fw
+install: snom-fw
+
+[pkg_345-fw]
+description: Firmware for Snom D345
+description_fr: Firmware pour Snom D345
+version: 10.1.119.10
+files: 345-fw
+install: snom-fw
+
+[pkg_385-fw]
+description: Firmware for Snom D385
+description_fr: Firmware pour Snom D385
+version: 10.1.119.10
+files: 385-fw
+install: snom-fw
+
+[pkg_712-fw]
+description: Firmware for Snom D712
+description_fr: Firmware pour Snom D712
+version: 10.1.119.10
+files: 712-fw
+install: snom-fw
+
+[pkg_715-fw]
+description: Firmware for Snom 715
+description_fr: Firmware pour Snom 715
+version: 10.1.119.10
+files: 715-fw
+install: snom-fw
+
+[pkg_717-fw]
+description: Firmware for Snom D717
+description_fr: Firmware pour Snom D717
+version: 10.1.119.10
+files: 717-fw
+install: snom-fw
+
+[pkg_725-fw]
+description: Firmware for Snom 725
+description_fr: Firmware pour Snom 725
+version: 10.1.119.10
+files: 725-fw
+install: snom-fw
+
+[pkg_735-fw]
+description: Firmware for Snom D735
+description_fr: Firmware pour Snom D735
+version: 10.1.119.10
+files: 735-fw
+install: snom-fw
+
+[pkg_785-fw]
+description: Firmware for Snom D785
+description_fr: Firmware pour Snom D785
+version: 10.1.119.10
+files: 785-fw
+install: snom-fw
+
+[pkg_uxm-fw]
+description: Firmware for Snom Extension USB Module UXM D3/D7
+description_fr: Micrologiciel pour module d'extension USB Snom UXM D3/D7
+version: 2.1.1
+files: uxm-fw
+install: snom-fw
+
+[install_snom-fw]
+a-b: cp $FILE1 firmware/
+
+[file_uxm-fw]
+url: http://downloads.snom.com/snomUXM-2.1.1.bin
```

Comment:
@ymolinet could you please use HTTPS links rather than HTTP ones, at least for `10.1.119.10` firmwares ?
HTTP links tend not to respond, whereas HTTPS ones flawlessly succeed.
HTTP to HTTPS modification was introduced in #136 a few weeks ago.

Also please take `uxmc` firmware ([this](https://github.com/wazo-platform/wazo-provd-plugins/blob/master/plugins/wazo-snom/10.1.101.11/pkgs/pkgs.db#L78-L83) and [this](https://github.com/wazo-platform/wazo-provd-plugins/blob/master/plugins/wazo-snom/10.1.101.11/pkgs/pkgs.db#L88-L91)), as also introduced in #136.

Many thx 👍

Reply:
@ymolinet many thanks for the HTTPS links 👍
Could you also take `uxmc` firmware as proposed above ?
Thank you again !